doc/ctc-meetings/2016-10-26.md

```diff
@@ -0,0 +1,151 @@
+# Node Foundation CTC Meeting 2016-10-26
+
+## Links
+
+* **Audio Recording**: TBP
+* **GitHub Issue**: 
+[#9261](https://github.com/nodejs/node/issues/9261)
+* **Minutes Google Doc**: <https://docs.google.com/document/d/1dUPFNyK8Yn1yWRAcXPY-mWM2KMWvOOB8Z5BH2s7RpJs>
+* _Previous Minutes Google Doc: <https://docs.google.com/document/d/18rnF8IfO10aQ0gRRasH3zSuv397-lm_x1fX1FSGcV7E>_
+
+
+## Present
+
+* Anna Henningsen @addaleax (CTC)
+* Bradley Meck @bmeck (observer/GoDaddy/TC39)
+* Ben Noordhuis @bnoordhuis (CTC)
+* Сковорода Никита Андреевич @ChALkeR (CTC)
+* Colin Ihrig @cjihrig (CTC)
+* Evan Lucas @evanlucas (CTC)
+* James M Snell @jasnell (CTC)
+* Brian White @mscdex (CTC)
+* Ali Ijaz Sheikh @ofrobots (CTC)
+* Seth Thompson @s3ththompson (observer/Google)
+* Shigeki Ohtsu @shigeki (CTC)
+* Sakthipriyan Vairamani @thefourtheye (observer)
+* Rich Trott @Trott (CTC)
+
+
+## Standup
+
+* Anna Henningsen @addaleax (CTC)
+* Not much
+* Bradley Meck @bmeck (observer/GoDaddy/TC39)
+* Work on making inspector work w/ vm
+* Minor talks w/ modules spec authors
+* Ben Noordhuis @bnoordhuis (CTC)
+* 
+* Сковорода Никита Андреевич @ChALkeR (CTC)
+* Rebuilt a new dataset from npm packages, some further work on the tooling
+* Some issue/pr comments as usual
+* Some ecosystem security stuff
+* Colin Ihrig @cjihrig (CTC)
+* Reviewing issues and PRs, opened a few PRs, libuv 1.10.0 update
+* Evan Lucas @evanlucas (CTC)
+* Working on getting libuv to use fsevents for file watching on OS X
+* Opened small PR to fix a test that kept failing on freebsd
+* James M Snell @jasnell (CTC)
+* Getting v7.0.0 out the door
+* PRs
+* More work on HTTP/2
+* Brian White @mscdex (CTC)
+* Continued working on string encoding/decoding performance. Starting to see even more promising results.
+* Reviewed PRs, commented on issues
+* Ali Ijaz Sheikh @ofrobots (CTC)
+* Not much
+* Seth Thompson @s3ththompson (observer/Google)
+* V8 5.5 beta shipped with async/await
+* Shigeki Ohtsu @shigeki (CTC)
+* Reviewed a few PR and made a security assessments of CVE-2016-8610 for Node-v0.10 and 0.12.
```

Comment:
@shigeki and your conclusion is? was Node vulnerable before the patch in openssl 1.0.2.j? /cc @mhdawson